etc/docker/demo/rucio.cfg

```diff
@@ -182,8 +182,8 @@ dataset_size = 5000000000
 dataset_dir = ~/public/sonar_medium_dataset/
 scope = user.vzavrtan
 
-[permission]
-policy=demo
+[policy]
+permission=demo
```

Comment:
```[policy]
permission = generic
schema = generic
support = https://github.com/rucio/rucio/issues/
support_rucio = https://github.com/rucio/rucio/issues/
```

Reply:
Right, actually it should be the generic permission. @hahahannes can you please submit a second PR with this 😄 thanks